### PR TITLE
Add some documentation on post-release TODO's

### PR DESCRIPTION
**Pull request by rramkumar1**
This is mainly for keeping track of things we need to do when a new release is cut. 

/assign @nikhiljindal

### CODE REVIEW

File: docs/dev/post-release-TODOS.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Post-Release TODO's
+
+This document explains steps that need to be taken after a new release
+of the ingress-gce controller has been cut and pushed.
+
+## Update Manifest
+
+The glbc.manifest in the main kubernetes repository needs to be updated
+to use the new image. The file is located at kubernetes/cluster/gce/manifests/glbc.manifest.
```
Comment (nikhiljindal) [inline docs/dev/post-release-TODOS.md]: Add a link to https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/manifests/glbc.manifest

File: docs/dev/post-release-TODOS.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Make sure to not only update the image for the container but also update the
+name in the top-level metadata field as well as version field under metadata.labels.
```
Comment (nikhiljindal) [inline docs/dev/post-release-TODOS.md]: Will linking to an example PR help?

File: docs/dev/post-release-TODOS.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Update e2e Tests  
+
+Our e2e tests need to be updated in order to make use of the new release.
+
+### ci-ingress-gce-upgrade-e2e
+
+Find a file called config.json under test-infra/jobs/config.json. In this file,
```
Comment (nikhiljindal) [inline docs/dev/post-release-TODOS.md]: Same comment as above, add a link to that file and an example PR

File: docs/dev/post-release-TODOS.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+find the json block called `ci-ingress-gce-upgrade-e2e`. In this block, modify
+the environment variable `GCE_GLBC_IMAGE` to point to the latest release image.
+
+### ci-ingress-gce-downgrade-e2e
+
+Find a file called nodes_util.go under kubernetes/test/e2e/framework.
```
Comment (nikhiljindal) [inline docs/dev/post-release-TODOS.md]: same comment as above :)

File: docs/dev/post-release-TODOS.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Post-Release TODO's
```
Comment (nikhiljindal) [inline docs/dev/post-release-TODOS.md]: optional nit: TODO's have a specific meaning in code and I dont think thats what we mean here.
Some other naming suggestions: release-guide or release-steps.

Feel free to ignore the comment.

File: docs/dev/post-release-steps.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+[Example PR](https://github.com/kubernetes/kubernetes/pull/62075)
+
+## Update e2e Tests
+
+Our e2e tests need to be updated in order to make use of the new release.
+
+### ci-ingress-gce-upgrade-e2e
+
+In this [file](https://github.com/kubernetes/test-infra/blob/master/jobs/config.json),
```
Comment (nikhiljindal) [inline docs/dev/post-release-steps.md]: nit: its generally a good idea to print the name of the file, so `In [config.json]` instead of `In this [file]`

**Comment by nikhiljindal** [comment]
Thanks for the fixes @rramkumar1 
I have one minor comment.
/lgtm

File: docs/dev/post-release-steps.md (hunk 6 above)
Comment (rramkumar1) [reply docs/dev/post-release-steps.md]: Done

**Pull request merged**